plugins/nitrokey/meson.build

```diff
@@ -1,4 +1,5 @@
-if get_option('plugin_nitrokey')
+if get_option('plugin_nitrokey').auto() or \
+   get_option('plugin_nitrokey').enabled()
```

Comment:
This too seems like a situation where you want a boolean option (yes/no to build it) rather than a tristate (in the auto case, only build it when dependencies are autodetected).


What is the purpose of auto, here? Is it literally identical to enabled?

Reply:
Yeah, I understand much better now, thanks for explaining it.  That was my initial inclination that it should have stayed boolean.  I'll drop the commits for the things that should really stay boolean.